### PR TITLE
Add changelog for version 3.1

### PR DESCRIPTION
**Pull request by cornelinux**

### CODE REVIEW

**Comment by cornelinux** [comment]
It was difficult to me to clarify which issue was a fix or an enhancement.
And which issue actually was an enhancement or a feature?
I tried to order the issues. But I am not that content with it.

File: Changelog [hunk 1]
```diff
@@ -1,3 +1,67 @@
...
+  * User experience:
+    * Improve user experience when enrolling Yubikeys via ykpersonalize - Automatically 
+      removing whitespaces (#1735)
+    * Allow user to change the token description (#1717)
+    * Customize Web UI page title (#1624, #1243)
+    * *search_on_enter* also applies to audit log (#1493)
+    * Allow a welcome message in the Web UI if the user has not token (#1074)
+  * Management:
+    * Add script to export OTP counters (#1728)
+    * Admins can only see tokens within the realm they are allowed to manage (#1713)
```
Comment (fredreichbier) [inline Changelog]: I think we could promote this to a main feature :)

File: Changelog [hunk 2]
```diff
@@ -1,3 +1,67 @@
...
+    * Allow many additional tags in email notifications: serial, user, givenname,
+      surname, username, userrealm, tokentype, recipient_givenname, recipient_surname,
+      time, date (#1703)
+    * Improve diagnostics script by adding SQLAlchemy URL (#1667)
+    * Add resolver conditions to several policy checks (#1646)
+    * /auth entries in the audit log now also fill in resolver and serial (#1593)
+    * `pi-manage backup` also backs up the FreeRADIUS configuration (#1575)
+    * Allow event handlers on /auth endpoint (#1567)
+    * Add adminread policies for policies, events, resolvers, system, machineresolvers,
```
Comment (fredreichbier) [inline Changelog]: I think this could also be a main feature.

File: Changelog [hunk 3]
```diff
@@ -1,3 +1,67 @@
...
+    * Endpoint GET /token/challenges also takes transaction_id (#1689)
+    * Push token can delay the response of /validate/check, so that there is no need 
+      to query the server to check if the push notification has been answered (#1583)
+  * User experience:
+    * Improve user experience when enrolling Yubikeys via ykpersonalize - Automatically 
+      removing whitespaces (#1735)
+    * Allow user to change the token description (#1717)
+    * Customize Web UI page title (#1624, #1243)
+    * *search_on_enter* also applies to audit log (#1493)
+    * Allow a welcome message in the Web UI if the user has not token (#1074)
```
Comment (fredreichbier) [inline Changelog]: typo: *no

File: Changelog (hunk 1 above)
Comment (fredreichbier) [inline Changelog]: Also, could you add a note that a migration policy ``pi-update-policy-b9131d0686eb`` is added?

File: Changelog [hunk 4]
```diff
@@ -1,3 +1,67 @@
...
+    * New policy *max_active_tokens_per_user* (#1241)
+    * Add image url to the otpauth QR code, allow images in e.g. FreeOTP (#1228)
+  * Performance:
+    * Make the serverpool in LDAP resolver persistant improving redundancy performance (#1396)
+
+  Fixes:
+  * Improve the stability of the schema-update-script (#1760)
+  * Rearrange update order in migration scripts (#1733)
+  * Adapt privacyidea-token-janitor to run with the TokenOwner table (#1709)
+  * Reordering decorators and policy checks to avoid unnesseray error messages (#1751)
```
Comment (fredreichbier) [inline Changelog]: typo: *unnecessary

File: Changelog [hunk 5]
```diff
@@ -1,3 +1,67 @@
+Version 3.1, 2019-09-XX
```
Comment (fredreichbier) [inline Changelog]: Sorry, I only now noticed we're missing some issues that had no milestone:
#1789
#1782
#1707
#1663

Comment (cornelinux) [reply Changelog]: Oups, some of them were so small that I was not sure, what to write (TiQR) and some obviously missed the milestone tag (I was going via the issues in the milestone)

But will do.

File: Changelog (hunk 4 above)
Comment (cornelinux) [reply Changelog]: done in  7d9d519c

File: Changelog (hunk 1 above)
Comment (cornelinux) [reply Changelog]: done in  7d9d519c

File: Changelog (hunk 3 above)
Comment (cornelinux) [reply Changelog]: done in  7d9d519c

File: Changelog (hunk 2 above)
Comment (cornelinux) [reply Changelog]: done in  7d9d519c

File: Changelog (hunk 1 above)
Comment (cornelinux) [reply Changelog]: done in  7d9d519c

**Pull request merged**